src/components/Diet/LightBox.js

```diff
@@ -5,19 +5,23 @@ import { getFoods } from '../../selectors/foods/getFoods';
 export const LightBox = ({ lightBoxData }) => {
   const foods = useSelector(getFoods);
 
-  return lightBoxData && lightBoxData.map((d) => {
-    const foodName = foods[d[0].food].desc;
-    const {skipGrams} = foods[d[0].food];
-    const foodQtty = d[0].qtty;
+  return lightBoxData && lightBoxData.map((data) => {
+    const foodName = foods[data[0].food].desc;
```

Comment:
just a picky comment. This is my bad anyway
`lightBoxData = [meal, similarFoods]`
we can destructure this using lightBoxData.map(([meal, similarFoods]) and then replace data[0] by `meal` and data[1] by `similarFoods`

Reply:
Agree. Cleaner and more readable. Please check next commit thoroughly